Win11ThemeTest/CalendarTest.cs

```diff
@@ -0,0 +1,269 @@
+            nextBtn.Click();
+        }
+
+        //test for click next year button
+        [Test]
+        public void cal7_isCalendarClickNextYear()
+        {
+            Assert.IsNotNull(calendar);
+            headerBtn = calendar.FindFirstChild(cf => cf.ByAutomationId("PART_HeaderButton"));
+            nextBtn = calendar.FindFirstChild(cf => cf.ByAutomationId("PART_NextButton"));
+            nextBtn.Click();
+            string headerName = headerBtn.Name;
+            DateTime nextYear = DateTime.Today.AddYears(1);
+            int year = nextYear.Year;
+            Assert.That(Convert.ToInt32(headerName), Is.EqualTo(year));
+            //reset
+            prevBtn = calendar.FindFirstChild(cf => cf.ByControlType(ControlType.Button));
+            prevBtn.Click();
+        }
+
+        //test for click year button
+        [Test]
+        public void cal8_isCalendarClickYear()
+        {
+            Assert.IsNotNull(calendar);
+            headerBtn = calendar.FindFirstChild(cf => cf.ByAutomationId("PART_HeaderButton"));
+            headerBtn.Click();
+            string headerName = headerBtn.Name;
+            DateTime currentMonth = DateTime.Today;
+            int year = currentMonth.Year;
+            string[] parts = headerName.Split('-');
+            int yearLow = Convert.ToInt32(parts[0]);
+            int yearHigh = Convert.ToInt32(parts[1]);
+            Assert.That(year, Is.GreaterThan(yearLow));
+            Assert.That(year, Is.LessThan(yearHigh));
+        }
+
+        //test for click previous year range button
+        [Test]
+        public void cal9_isCalendarClickPrevYearRange()
+        {
+            Assert.IsNotNull(calendar);
+            headerBtn = calendar.FindFirstChild(cf => cf.ByAutomationId("PART_HeaderButton"));
+            string headerName = headerBtn.Name;
+            prevBtn = calendar.FindFirstChild(cf => cf.ByControlType(ControlType.Button));
+            prevBtn.Click();
+            string prevHeaderName = headerBtn.Name;
+            DateTime currentMonth = DateTime.Today;
+            int year = currentMonth.Year;
+            string[] parts = headerName.Split('-');
+            int yearLow = Convert.ToInt32(parts[0]);
+            int yearHigh = Convert.ToInt32(parts[1]);
+            string[] prevParts = prevHeaderName.Split('-');
+            int prevYearLow = Convert.ToInt32(prevParts[0]);
+            int prevYearHigh = Convert.ToInt32(prevParts[1]);
+            int lowDiff = yearLow - prevYearLow;
+            int highDiff = yearHigh - prevYearHigh;
+            Assert.That(lowDiff, Is.EqualTo(10));
+            Assert.That(highDiff, Is.EqualTo(10));
+            //reset
+            nextBtn = calendar.FindFirstChild(cf => cf.ByAutomationId("PART_NextButton"));
+            nextBtn.Click();
+        }
+
+        //test for click next year range button
+        [Test]
+        public void cald1_isCalendarClickPrevYearRange()
```

Comment:
check when this test is running individually is it running correctly, is it failing with exception - System.FormatException : The input string 'February 2024' was not in a correct format.